instances/geneticStrainType/knockin.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
all "null"s are marked in red for some reason... you can double check but I think nothing is wrong...

Reply:
Yeah, I saw those too. Can't see anything wrong either. Let merge it like this and see if it will be a problem?